src/components/Receiving/ReceivingHistory.js

```diff
@@ -54,30 +55,35 @@ class ReceivingHistory extends Component {
   constructor(props) {
     super(props);
     this.callout = React.createRef();
+
+    this.receivingHistoryOffset = 0;
   }
 
   state = {
     checkedPiecesMap: {},
     confirming: false,
     isAllChecked: false,
-    isLoading: true,
     pieces: [],
     searchText: '',
   };
 
   componentDidMount() {
+    const { mutator } = this.props;
+
+    mutator.receivingHistory.reset();
     this.fetchHistory();
   }
 
   fetchHistory = () => {
     const { mutator, match: { params: { id, lineId } } } = this.props;
+    const { searchText = '' } = this.state;
```

Comment:
in state initialization `searchText: '',` , I'm not sure if it could be rather than empty string

Reply:
right, forgot to remove, thanks